packages/frontend-2/components/form/Button.vue

```diff
@@ -104,31 +105,44 @@ const colorClasses = computed(() => {
   switch (props.type) {
     case 'outline':
       return `${
-        isDisabled ? 'text-foreground-3' : 'text-foreground'
-      } bg-transparent border-foreground-3 focus:ring-primary-lighter`
+        isDisabled
+          ? 'bg-neutral-300 dark:bg-neutral-800 text-neutral-100 dark:text-neutral-400 focus:ring-0 hover:ring-0'
+          : 'text-primary'
+      } border-2 border-primary ring-background-accent`
     case 'danger':
       return `${
-        isDisabled ? 'bg-danger-darker' : 'bg-danger'
-      } text-white hover:bg-danger-darker focus:ring-danger-lighter`
-    case 'success':
-      return `${
-        isDisabled ? 'bg-success-darker' : 'bg-success'
-      } text-white hover:bg-success-darker focus:ring-success-lighter`
+        isDisabled
+          ? 'bg-neutral-300 dark:bg-neutral-800 text-neutral-100 dark:text-neutral-400 focus:ring-0 hover:ring-0'
+          : 'bg-danger'
+      } text-white hover:bg-danger-darker ring-danger-lighter`
     case 'warning':
       return `${
-        isDisabled ? 'bg-warning-darker' : 'bg-warning'
-      } text-white hover:bg-warning-darker focus:ring-warning-lighter`
-    case 'secondary':
+        isDisabled
+          ? 'bg-neutral-300 dark:bg-neutral-800 text-neutral-100 dark:text-neutral-400 focus:ring-0 hover:ring-0'
```

Comment:
Ideally you shouldn't need to use `dark:` at all - semantic colors would be set up in such a way that they look correct in both themes. Maybe it makes sense to have this be an exception, but I just wanted to note this so that we're on the same page about this being something that we shouldn't need to do often.